internal/cli/kubernetes/config/apply.go

```diff
@@ -105,7 +105,9 @@ func (opts *ApplyOpts) Run() error {
 		WithTargetOperatorVersion(opts.operatorVersion).
 		WithSecretsData(true).
 		WithFeatureValidator(atlasCRDs).
-		WithPatcher(atlasCRDs)
+		WithPatcher(atlasCRDs).
+		WithDataFederationNames(opts.dataFederationName).
```

Comment:
I am guessing we were not applying data federation names before?

Reply:
No. The goal of this PR is exactly add these 2 options (dataFederationNames and independentResources).